src/lib.rs

```diff
@@ -162,3 +208,371 @@ impl LevelIndex {
         }
     }
 }
+
+bitflags::bitflags! {
+    #[repr(transparent)]
+    pub struct ChannelTypeQualifiers: u32 {
+        const LINEAR        = (1 << 0);
+        const EXPONENT      = (1 << 1);
+        const SIGNED        = (1 << 2);
+        const FLOAT         = (1 << 3);
+        const NONE          = 0;
+        const UNINITIALIZED = 0xFFFF;
```

Comment:
What's the purpose of these constants? They don't seem to be present in the DFD spec, afaict.

Reply:
No indeed they aren’t in the spec. They’re a pattern from somewhere and I’ve copied, pasted, and modified. I’m yet to find a use for uninitialised, but none is useful, albeit redundant. I’ll remove them.

Reply:
Removed.